internal/integrations/v4/executor/config.go

```diff
@@ -10,8 +10,9 @@ import (
 
 // Config describes the context to execute a command: user, directory and environment variables.
 type Config struct {
-	User      string
-	Directory string
+	User         string
+	Directory    string
+	InstanceName string
```

Comment:
I would call it `IntegrationName`. It's consistent with  the log field and also I think that instance can mislead to a specific integration instance :

integration_name: `nri-nginx`
instance_name: `localhost:8081`

wdyt?

Reply:
you're right, I thought this is a configurable name that is given to an integration instance, but I was wrong. I'll correct this